src/Microsoft.Data.Sqlite/SqliteFactory.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Data.Common;
+
+namespace Microsoft.Data.Sqlite
+{
+    public class SqliteFactory : DbProviderFactory
+    {
+        static SqliteFactory()
+        {
+            Instance = new SqliteFactory();
+        }
+
+        private SqliteFactory()
+        {
+        }
+
+        public static SqliteFactory Instance { get; }
```

Comment:
The old EF stack looks for the Instance field. At some point I'm pretty sure I updated it to look for a property if the field wasn't found. But if we want to be as compatible as possible with this convention, then we should use a field.